_snippets/graphql/remove-members-from-tier.mdx

```diff
@@ -25,8 +25,7 @@ mutation removeMembersFromTier($input: RemoveMembersFromTierInput!) {
       {
         "companyId": "co_123"
       }
-    ],
-    "tierId": "tier_123"
+    ]
```

Comment:
I don't understand - why are we no longer specifying which tiers I'm removing a member from? How does this work?

Reply:
a tenant/company can only be part of a single tier. The ID of the entity is sufficient to remove it from the tier it's part of

Reply:
Amazing - makes sense thanks.